### PR TITLE
- Added travis-ci build files

### PR DESCRIPTION
**Pull request by antoinecampbell**
I don't have admin rights to enable travis-ci builds for this repo, I'll need admin access or someone else can just enable it. Be sure to  enable the option to only build in .travis.yml is present. Issue #21

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - oraclejdk8
+android:
+  components:
+    # Uncomment the lines below if you want to
+    # use the latest revision of Android SDK Tools
+    - platform-tools
+    - tools
+
+    # The BuildTools version used by your project
+    - build-tools-24.0.0
```
Comment (OllenDev) [inline .travis.yml]: why are both build-tools enabled?

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (OllenDev) [inline build.sh]: Could we combine the script into the travis file or move the before script steps into this file?

File: .travis.yml (hunk 1 above)
Comment (antoinecampbell) [inline .travis.yml]: One is for the emulator the other is for building the actual project. The emulator for android-24 won't run on the container being used. So I'm using an emulator I know works on the travis container

File: build.sh (hunk 2 above)
Comment (antoinecampbell) [inline build.sh]: I wouldn't want to put the contents of the build.sh into the travis.yml, but we could possibly put the contents of the before_script into the build.sh file. I'll give it a try and make sure it still build properly.

**Comment by antoinecampbell** [comment]
Moving the emulator management to the build.sh file worked, the build passed

**Comment by antoinecampbell** [comment]
@OllenDev I went back to look why I have two build-tools, the app module uses 24.0.1 while the ranger module uses 24.0.0

**Comment by rdrobinson3** [comment]
@antoinecampbell ive got some stuff coming soon that will bring them in sync. I will update the builds.

**Pull request merged**